content/en/news/releases/1.24.x/announcing-1.24/_index.md

```diff
@@ -2,7 +2,7 @@
 title: Announcing Istio 1.24.0
 linktitle: 1.24.0
 subtitle: Major Release
-description: Istio 1.24 Release Announcement.
+description: Istio 1.24 Release Announcement
```

Comment:
All the others have a period, not sure why

Reply:
I think it's a lint rule

Reply:
Yes, that made the build fail

> 10:44:57 AM: ERROR Page description doesn't end with a period: 'Istio 1.24 Release Announcement'